Plugins/JiraCommitHintPlugin/JiraCommitHintPlugin.cs

```diff
@@ -27,14 +27,17 @@ public class JiraCommitHintPlugin : GitPluginBase, IGitPluginForRepository
         private static readonly TranslationString EmptyQueryResultCaption = new("First Task Preview");
 
         private const string DefaultFormat = "{Key} {Summary}";
-        private const string BASIC = "Basic";
-        private const string JWT = "JWT";
+        private const string AuthTypeBasic = "User name & Password";
+        private const string AuthTypeJwt = "Personal access token";
+
+        // It not a good idea use this user name for Personal access token, but credentials required.
+        private const string AuthTypeJwtUserName = "#Jira.Commit.Hint:access.token$";
         private Jira? _jira;
         private string? _query;
         private string _stringTemplate = DefaultFormat;
         private readonly BoolSetting _enabledSettings = new("Jira hint plugin enabled", false);
         private readonly StringSetting _urlSettings = new("Jira URL", @"https://jira.atlassian.com");
-        private readonly ChoiceSetting _authTypeSettings = new("Auth type", new List<string> { BASIC, JWT });
+        private readonly ChoiceSetting _authTypeSettings = new("Auth type", new List<string> { AuthTypeBasic, AuthTypeJwt });
```

Comment:
PAT first?

Reply:
OK

Reply:
I tried to PAT first, but got some accident with migrate. If user use username/password already, we think they sholudn't make any change to use it still. But if PAT first, they have to change their settings.